src/entries/popup/components/ExplainerSheet/ExplainerSheet.tsx

```diff
@@ -118,14 +118,13 @@ export const ExplainerSheet = ({
         active: false,
       });
   }, []);
-
   return (
     <BottomSheet
       onClickOutside={onClickOutside || actionButton?.action}
       zIndex={zIndexes.EXPLAINER_BOTTOM_SHEET}
       show={show}
     >
-      <Box testId={`explainer-sheet-${testId}`}>
+      <Box testId={`explainer-sheet-${testId}`} isExplainerSheet>
```

Comment:
isn't an explainer sheet a modal? is this new prop different than the `isModal` one?

Reply:
It is, but I think this is necessary in the case that there are nested BottomSheets (like ReviewSheet > ExplainerSheet)